go/cmd/internal/docgen/docgen.go

```diff
@@ -106,16 +189,19 @@ func frontmatterFilePrepender(filename string) string {
 		cmdName = root
 	}
 
+	cmdName = strings.ReplaceAll(cmdName, "_", " ")
+
 	return fmt.Sprintf(frontmatter, cmdName, root)
 }
 
 func linkHandler(filename string) string {
-	name := filepath.Base(filename)
-	base := strings.TrimSuffix(name, filepath.Ext(name))
+	base := filepath.Base(filename)
+	name := strings.TrimSuffix(base, filepath.Ext(base))
```

Comment:
these names ended up being more clear to me ("why is the thing we get from `.Base` not called base?")